gradio/components/dataframe.py

```diff
@@ -308,6 +308,15 @@ def postprocess(
                     "Cannot display Styler object in interactive mode. Will display as a regular pandas dataframe instead."
                 )
             df: pd.DataFrame = value.data  # type: ignore
+            hidden_cols = []
+            if hasattr(value, "hidden_columns"):
+                hidden_cols = value.hidden_columns  # type: ignore
+
+            visible_cols = [
+                i for i, col in enumerate(df.columns) if i not in hidden_cols
+            ]
```

Comment:
for conciseness, you could do something like this:
```suggestion
            visible_cols = [
                i for i, col in enumerate(df.columns) if i not in getattr(value, "hidden_columns", [])
            ]
```